vmdb/app/models/dialog_field_drop_down_list.rb

```diff
@@ -1,2 +1,55 @@
 class DialogFieldDropDownList < DialogFieldSortedItem
+  has_one :resource_action, :as => :resource, :dependent => :destroy
+
+  after_initialize :default_resource_action
```

Comment:
Yeah, that was the only reason why I didn't include it in the base class, I figured it would get changed on the last one, but we could change it now if you want.